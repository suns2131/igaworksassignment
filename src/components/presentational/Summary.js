import React from "react";

function Summary({ name }) {
  return (
    <div>
      <h4>{name}</h4>
    </div>
  );
}

export default Summary;
