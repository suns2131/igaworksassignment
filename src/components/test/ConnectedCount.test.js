/* eslint-disable no-undef */
import React from "react";
import { render } from "@testing-library/react";
import Summary from "../presentational/Summary";

it("matches snapshot", () => {
  const utils = render(<Summary current={41233} subtraction={93} description="Unique Event Count" />);
  expect(utils.container).toMatchSnapshot();
});
