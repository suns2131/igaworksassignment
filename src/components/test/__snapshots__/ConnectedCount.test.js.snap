// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="sc-bczRLJ iNMQBj"
  >
    <div
      class="DescDiv"
    >
      <span
        class="sc-gsnTZi kIhmbF"
      >
        SUM
      </span>
      <div>
        Unique Event Count
      </div>
    </div>
    <div
      class="Sums"
    >
      41,233
    </div>
    <div
      class="Plus"
    >
      â†‘
      93
    </div>
  </div>
</div>
`;
